.github/workflows/build.yml

```diff
@@ -177,8 +178,17 @@ jobs:
   test:
     needs:
       - diagnostics
+    permissions:
+      # actions/checkout needs this to fetch code
+      contents: read
     runs-on: ubuntu-latest
     steps:
+      # Note that a duplicate of this step must be added at the top of
+      # each job.
```

Comment:
```suggestion
```

Reply:
Resolved in [5a02fb8](https://github.com/cisagov/skeleton-packer/pull/376/commits/5a02fb8244d9bfb8d1a37cd1e2e3deb60bdf03cc).